proposals/PROPOSALS.md

```diff
@@ -0,0 +1,156 @@
+A HIP (*Haskell Foundation Initiative Proposal*) is a process for submitting a proposal for an initiative to be undertaken by the Haskell Foundation (HF). The main motivation of this document is to detail the primary mechanism through which the community may propose, discuss and implement initiatives to be carried out through the Haskell Foundation that will affect the Haskell ecosystem. For each HIP, all changes to the ecosystem go through a proposal process, which are is discussed by the HF Technical Track, the broader community, and any and all interested stakeholders. Only upon reaching a consensus are initiatives accepted to be merged and executed.
```

Comment:
(Wordsmithing.)  

This document describes how the community may propose....

Reply:
Added